components/match2/wikis/valorant/big_match.lua

```diff
@@ -37,7 +37,12 @@ function BigMatch.run(frame)
 	-- Don't store match1 as BigMatch records are not complete
 	Match.store(match, {storeMatch1 = false, storeSmw = false})
 
-	local tournamentData = bigMatch:_fetchTournamentInfo(args.tournamentlink or '')
+	-- Attempty to automatically retrieve tournament link from the bracket
+	if Logic.isEmpty(args.tournamentlink) then
```

Comment:
```suggestion
	if String.isEmpty(args.tournamentlink) then
```

Right?

Reply:
Technically doesn't matter as Logic.isEmpty and StringUtils.isEmpty work the same, except Logic has a special provision to handle tables in the input. However I agree it's easier to read.

Reply:
Yes I'm aware, mostly as you say for readability.